### PR TITLE
Making test-session standalone and runnable on localhost

### PR DESCRIPTION
**Pull request by alessandrothea**
- Added connectivity server application
- Removed application name from opmon service app to make it non-runnable

### CODE REVIEW

**Comment by alessandrothea** [comment]
Requires 
1. https://github.com/DUNE-DAQ/confmodel/pull/33
2. https://github.com/DUNE-DAQ/drunc/pull/128

File: test/config/test-session.data.xml [hunk 1]
```diff
@@ -104,9 +104,42 @@
...
+  <data val="-b 0.0.0.0:5000 --workers=1 --worker-class=gthread --threads=2 --timeout 5000000000 --log-level=debug connection-service.connection-flask:app"/>
+ </attr>
+ <attr name="threads" type="u16" val="1"/>
+ <rel name="runs_on" class="VirtualHost" id="vlocalhost"/>
+ <rel name="exposes_service">
+  <ref class="Service" id="local-connectivity-service"/>
+ </rel>
+</obj>
+
+<obj class="ConnectionService" id="local-connection-server-works">
```
Comment (gcrone) [inline test/config/test-session.data.xml]: Is this one just left over from debugging?

File: test/config/test-session.data.xml [hunk 2]
```diff
@@ -324,10 +354,13 @@
  <rel name="disabled">
   <ref class="DetectorToDaqConnection" id="det-4-connections"/>
   <ref class="DetectorToDaqConnection" id="det-3-connections"/>
+  <ref class="Segment" id="df-segment"/>
+  <ref class="Segment" id="trg-segment"/>
```
Comment (gcrone) [inline test/config/test-session.data.xml]: Are thse intentionally in the disabled list?

**Review by gcrone — APPROVED** [review]
Looks good but I had a couple of comments

**Comment by alessandrothea** [comment]
Thanks @gcrone, fixed and pushed

**Pull request merged**